test/cppsuite/tests/poc_test.cxx

```diff
@@ -31,4 +31,13 @@
 class poc_test : public test_harness::test {
```

Comment:
Why we don't just implement the operation as part of the test and pass the test itself as a reference?

Reply:
This is actually a good idea, I'll give it a go